src/bokeh/models/annotations/labels.py

```diff
@@ -161,6 +186,28 @@ def __init__(self, *args, **kwargs) -> None:
     distance in |screen units| from a given data position.
     """)
 
+    angle = Angle(default=0, help="""
+    The angle to rotate the text, as measured from the horizontal.
+    """)
+
+    angle_units = Enum(AngleUnits, default="rad", help="""
+    Acceptable values for units are ``"rad"`` and ``"deg"``.
+    """)
+
+    direction = Enum(Direction, default="anticlock", help="""
+    The direction of rotation of the label.
+
+    .. note::
+        This property is experimental and may change at any point.
+    """)
+
+    editable = Bool(default=False, help="""
```

Comment:
Would `rotatable` be better / more specific? When I opened this PR and saw "editable" I first thought this was a about editing the text of the labels

Reply:
This indeed may be confusing for labels, but is consistent with other APIs. All APIs broadly related to editability are experimental, so we can iterate over the design. Perhaps a better choice would have been `interactive`, but if we decide so, then this will have to be applied consistently. I will leave this for now. There will be more changes like this one in near future, specifically text editability of `Text` glyph and `Label`, so will have a better picture then.